sw/airborne/arch/chibios/mcu_periph/i2c_arch.c

```diff
@@ -345,8 +345,14 @@ void i2c3_hw_init(void)
 
 #if USE_I2C4
 PRINT_CONFIG_VAR(I2C4_CLOCK_SPEED)
+
+#if defined(STM32H7XX)
+// Local variables (in DMA safe memory)
+static IN_BDMA_SECTION(struct i2c_init i2c4_init_s) = {
```

Comment:
is this true for all H7 boards ?

Reply:
yes

Reply:
yes I2C4 and SPI6 are peripherals linked to BDMA on all STM32H7 whatever the card that use them.
the rational behind the use of I2C4 and SPI6 on the tawakiV2 is to migrate maximum number of peripherals from DMA to BDMA to free DMA channels that we want to later use with BIDIR DSHOT which need N+1 DMA channels for each timer using Bidir DSHOT, so up to 10 DMA channels for octocopter.